README.md

```diff
@@ -1,5 +1,213 @@
+USER STORY 10: Strike Bonus 1
+
+As a bowler  
+So that I can receive bonus points for a strike  
+My current frame score will be added to my previous frame score as bonus points, if I don't roll a strike in my current frame
+
+USER STORY 11: Strike Bonus 2
+
+As a bowler  
+So that I can receive bonus points for a strike  
+My current frame score and my first roll in my next frame will be added to my previous frame score as bonus points, if i roll a strike in my current frame
+
+USER STORY 12: Spare Bonus
+
+As a bowler  
+So that I can receive bonus points for a spare  
+The first roll score of my current frame will be added to my previous frame score as bonus points
+
+USER STORY 13: Frame 10, Roll 1 = Strike
+
+As a bowler  
+So that I can receive bonus points for a strike in my last frame (frame 10)  
+Allow me to have 2 additional rolls
+
+USER STORY 14: Frame 10 & Frame 9 Strikes
+
+As a bowler  
+So that I can receive bonus points for a strike in my last frame (frame 10) and the previous frame (frame 9)  
+Allow me to have 2 additional rolls and add to frame 10 and frame 9 accordingly
+
+USER STORY 15: Frame 10 Spare
+
+As a bowler  
+So that I can receive bonus points for a spare in my last frame (frame 10)  
+Allow me to have 1 additional roll  
+
+### APPROACH
+------------
+
+* Focused on understanding the game logic in detail, breaking down examples given to ensure understanding of bonus points and how the final score is calculated
+* Researched in detail how to access data in arrays and how to iterate over nested arrays in order to return a total score - sought help from peers which led me to the .flat() function
+* Broke all requirements down into simple user stories and tested each one of them
+* Attempted to keep each function simple and relative to one task only
+
+### FOLLOW UP
+------------
+
+###### POSSIBLE CHANGES TO LOGIC:
+* Need to split the BowlingScorecard constructor (class) into 2 in order to keep it simple - possibly BowlingFrame and BowlingScorecard
+* Tests are long due to the need to test last frame - possibly reduce number of tests to avoid repetition
+* Add in constants for minimum and maximum of knocked down pins to avoid use of magic numbers within functions
+* Add in private method syntax   
+  e.g. \_isLastFrameStrike
+* Add in error messages if the numbers entered for knockedDownPins in the rollScore function and the bonusRolls in the lastFrame functions are not between 1 and 10
+* Prevent more than 2 rollScores being added to a frame
+* Prevent more than 10 frames being added to total
+* cumScore does not include bonus points until the next frame(s) has been played - if a strike or spare has been scored, not sure if this should be blank in the logic or just in the user interface
+
+###### ADDITIONAL FEATURES:
+* Use jQuery, html & CSS to add an interactive user interface
+* Add in Travis to gain test coverage
+* Add in ESLint to check javaScript conventions are met
+
+
+ORIGINAL README: Bowling Challenge
```

Comment:
Excellent README file. It shows that you have thought through the process and limitations of your current code.